src/Pokemon.java

```diff
@@ -0,0 +1,77 @@
+public abstract class Pokemon {
+
+    public String getName() {
+        return name;
+    }
+
+    public int getLevel() {
+        return level;
+    }
+
+    public int getHp() {
+        return hp;
+    }
+
+    public String getFood() {
+        return food;
+    }
+
+    public String getType() {
+        return type;
+    }
+
+    public String getSound() {
+        return sound;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public void setLevel(int level) {
+        this.level = level;
+    }
+
+    public void setHp(int hp) {
+        this.hp = hp;
+    }
+
+    public void setFood(String food) {
+        this.food = food;
+    }
+
+    public void setType(String type) {
+        this.type = type;
+    }
+
+    public void setSound(String sound) {
+        this.sound = sound;
+    }
+
+    private String name;
```

Comment:
Je klasse variabelen mogen bovenaan, nog boven de methodes. 
De conventie is: 
- constanten
- variabelen
- constructors
- getters/setters en dan andere methodes (of eerst andere methodes en dan getters/setters)